### PR TITLE
Issue #11163: Enforced filesize InputOneStatementPerLineMultiline

### PR DESCRIPTION
**Pull request by mahfouz72**
issue #11163:
force file size on InputOneStatementPerLineMultiline
OneStatementPerLine module is done

### CODE REVIEW

**Comment by Vyom-Yadav** [comment]
@mahfouz72 Thanks for the PR, please fix the CI failures:
```
[ERROR] [checkstyle] [ERROR] /Users/runner/work/1/s/src/xdocs/releasenotes.xml:50: Line should not be longer than 100 symbols [lineLengthXml]
[ERROR] [checkstyle] [ERROR] /Users/runner/work/1/s/src/xdocs/releasenotes.xml:55: Line should not be longer than 100 symbols [lineLengthXml]
[INFO]      [echo] Checkstyle finished (checkstyle-checks.xml) : 28/01/2024 03:33:56 PM
[INFO]      [echo] Checkstyle started (checkstyle-non-main-files-checks.xml): 28/01/2024 03:32:20 PM
[INFO] [checkstyle] Running Checkstyle  on 4332 files
[ERROR] [checkstyle] [ERROR] /Users/runner/work/1/s/src/xdocs/releasenotes.xml:50: Line should not be longer than 100 symbols [lineLength]
[ERROR] [checkstyle] [ERROR] /Users/runner/work/1/s/src/xdocs/releasenotes.xml:55: Line should not be longer than 100 symbols [lineLength]
```

To run most of the CI tests locally, you can run `mvn clean verify` locally. See https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations for more details.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java
Comment (Vyom-Yadav) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java:76]: Please rever this minor change, if not required.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java
Comment (Vyom-Yadav) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java:52]: Please rever this minor change, if not required.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java
Comment (Vyom-Yadav) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java:29]: Please rever this minor change, if not required.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java
Comment (Vyom-Yadav) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java:41]: Please rever this minor change, if not required.

**Review by Vyom-Yadav — CHANGES_REQUESTED** [review]
Minor changes @mahfouz72 , rest lgtm

**Comment by mahfouz72** [comment]
@Vyom-Yadav these errors in CI is not related to my changes I didn't modify these files . they are probably were modifed before when updating the Release notes.I can help with fixing this issue. but what can I do with the line length to shorten it?

**Comment by Vyom-Yadav** [comment]
Hmm, the CI is broken @mahfouz72, thanks for pointing it out. I don't think a manual fix would suffice, but it would be awesome if you could create an issue about it.

**Comment by mahfouz72** [comment]
@Vyom-Yadav  can you check this PR #14373 I fixed the issue there. I don't know if this change is sufficient or not but you can take a look at it. and should I open an issue for this or just this PR is sufficient?

**Comment by Vyom-Yadav** [comment]
> should I open an issue for this or just this PR is sufficient?

Please create an issue for it, this should be fixed in the release notes generator.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (Vyom-Yadav) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java:72]: After some discussion, we cannot allow `test...One`, `test....Two` naming convention. It is better to logically split the input files so it is easier to debug and maintain it in the future.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (Vyom-Yadav) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java:87]: Same here, please change the class names to be descriptive of the content, so that we know what the test is testing.

**Review by Vyom-Yadav — CHANGES_REQUESTED** [review]
Items:

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (mahfouz72) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java]: @Vyom-Yadav done

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (mahfouz72) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java
Comment (mahfouz72) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java]: all is done

**Review by romani — APPROVED** [review]
Ok to merge after @Vyom-Yadav approval

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (Vyom-Yadav) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java:81]: From the #11163 Description:

> Example of update: https://github.com/checkstyle/checkstyle/commit/8673001d9b0edacfa0d54bec85b26db46d1cce4d
> 
> Test method names and input file name should be similar and describe what we are testing. Example from link above:
> 
> Test method name is testJavadocVariableOnInnerClassFields
> Input file name is InputJavadocVariableOnInnerClassFields.java

Please make the file name as descriptive as the test name.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (Vyom-Yadav) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java:94]: Same here. Please make the file name as descriptive as the test name.

**Review by Vyom-Yadav — CHANGES_REQUESTED** [review]
Items:

**Comment by Vyom-Yadav** [comment]
@mahfouz72 Please rebase onto the latest master.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (mahfouz72) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java]: @Vyom-Yadav done

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java
Comment (mahfouz72) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java]: @Vyom-Yadav done

**Comment by mahfouz72** [comment]
@Vyom-Yadav ping

**Comment by mahfouz72** [comment]
@romani @nrmancuso ping
sorry for constant pining but this is 2 weeks old. I think you won't notice it

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**
